app/src/main/java/org/mifos/mobile/ui/adapters/CheckBoxAdapter.kt

```diff
@@ -29,10 +29,13 @@ class CheckBoxAdapter @Inject constructor() :
     }
 
     override fun onBindViewHolder(holder: RecyclerView.ViewHolder, position: Int) {
-        val (status, color, isChecked) = statusList?.get(position)!!
+        var (status, color, isChecked) = statusList?.get(position)!!
         val states = arrayOf(intArrayOf(android.R.attr.state_checked), intArrayOf())
         val colors = intArrayOf(color, color)
         (holder as ViewHolder).cbStatusSelect?.isChecked = isChecked
+        holder.cbStatusSelect!!.setOnClickListener{
```

Comment:
The assertion operator `(!!)` throws a null pointer exception if `cbStatusSelect` is null, would be better if you use the safeCall operator `(?.)`

Reply:
I have done the changes ,sir